pymeasure/instruments/agilent/agilentB1500.py

```diff
@@ -175,6 +208,30 @@ def force_gnd(self):
         """
         self.write("DZ")
 
+    def restore_settings(self):
+        """Restores the settings of all channels to the state before
+        using `force_gnd`. (``RZ``)
```

Comment:
```suggestion
        using :meth:`force_gnd`. (``RZ``)
```
This :meth: will mark it as a method and link it (in the sphinx documentation) to the appropriate method.